### PR TITLE
Two ID edits

### PR DESCRIPTION
**Pull request by Gryph667**
Edited ID tzwws7 to reflect the updated canvas, removed ID tx1y4k as it was no longer present on the updated canvas.

### CODE REVIEW

File: web/atlas.json [hunk 1]
```diff
@@ -3333,7 +3333,6 @@
...
 {"id": "tx1y8s", "submitted_by": "slatourelle", "name": "Houdini", "description": "Houdini is a leading VFX and 3d software used for making VFX and tools for the games and film industries", "website": "https://www.sidefx.com/", "subreddit": "/r/houdini", "center": [1478.5, 1653.5], "path": [[1483.5, 1658.5], [1483.5, 1647.5], [1472.5, 1647.5], [1472.5, 1658.5]]},
-{"id": "tx1y4k", "submitted_by": "MineWarz", "name": "The Button", "description": "A reference to the popular 2015 Reddit April Fools' event 'The Button'. Made by the April Knights.", "website": "", "subreddit": "/r/thebutton", "center": [809.5, 672.5], "path": [[804.5, 670.5], [814.5, 670.5], [814.5, 673.5], [804.5, 673.5]]},
```
Comment (AnonymousRandomPerson) [inline web/atlas.json:3336]: Destroyed art is allowed. We are working on the ability to time-box them so they only appear at the proper points on the timeline.

File: web/atlas.json [hunk 2]
```diff
@@ -7156,7 +7155,7 @@
...
-{"id": "tzwws7", "submitted_by": "naliliV2", "name": "OptiFine", "description": "OptiFine is a Minecraft optimization mod. It allows Minecraft to run faster and look better with full support for high-definition textures and many configuration options.", "website": "https://optifine.net/home", "subreddit": "/r/Optifine", "center": [810.5, 669.5], "path": [[804.5, 664.5], [807.5, 664.5], [807.5, 661.5], [815.5, 661.5], [815.5, 677.5], [804.5, 677.5]]},
+{"id": "tzwws7", "submitted_by": "gryph667", "name": "Grandmaster Gryph", "description": "A depiction of a harpy eagle, the preferred profile pic of the Grandmaster of the April Knights during Place 2022.", "website": "https://aprilknights.org", "subreddit": "/r/Gryph667", "center": [810.5, 669.5], "path": [[804.5, 664.5], [807.5, 664.5], [807.5, 661.5], [815.5, 661.5], [815.5, 677.5], [804.5, 677.5]]},
```
Comment (AnonymousRandomPerson) [inline web/atlas.json:7158]: The previous entry is valid as it was present on an earlier canvas version. Post the new entry on Reddit to add it in.

**Review by AnonymousRandomPerson — CHANGES_REQUESTED** [review]
Address comments.

**Comment by Gryph667** [comment]
If there will be time gating, then this is a dead PR, as it needs a new entry, which would be on the sub

**Comment by Gryph667** [comment]
Yep, killing it.

**Pull request closed without merging**